### PR TITLE
Project review

### PR DESCRIPTION
**Pull request by koomekevo**
Project review and project changes

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,464 @@
...
+            </ul>
+            <!-- Links -->
+
+          </div>
+          <!-- Collapsible content -->
+
+
+          <div class="welcome-text col-12 pt-5 mt-5">
+            <h3 class="text-left red">"Hello Chess Fans!"</h3>
+            <h1 class="image-effect text-uppercase text-left font-weight-bold red">Online Chess Conference</h1>
```
Comment (afanimartin) [inline index.html]: This font is too large compared with the original design.
### Original design
original-hero

### Your design
review-hero

Als o pay attention to the left margin and the start and end dates for the conference. Please make the necessary changes and send for another review.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,464 @@
...
+    <footer class="container-fluid p-4 bg-light">
+      <div class="row">
+        <div class="col-6">
+          <figure>
+            <img class="img-fluid" src="./assets/images/creative.jpg" alt="Creative Common">
+          </figure>
+        </div>
+        <div class="col-6">
+          <p>2021 Creative Commons Korea <br>
+          Some Rights Reserved </p>
```
Comment (afanimartin) [inline index.html:457]: ### Original design
original-footer

### Your design
review-footer

The size of the Creative Commons differ a lot and there's so much space between the logo and the text in your case and the elements are not centered like in the given design. Please make the requested changes

**Review by afanimartin — CHANGES_REQUESTED** [review]
Hi @koomekevo, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

### Martin Afani
#### Slack: whiz25

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (koomekevo) [reply index.html]: Hello Afani,

Thanks for your review :recycle: I have managed to reduce the size of the font on the jumbotron section and made changes to the margin. However I am not able to insert the dates as zetafonts don't allow input of 0 - 9 on their free trial of cocogoose font https://www.zetafonts.com/supportold&mvc=AUD.

File: index.html (hunk 2 above)
Comment (koomekevo) [reply index.html:457]: I have these changes too thank you for highlighting :+1:

**Review by carlos-ssh — APPROVED** [review]
# 🍾🍾 **STATUS APPROVED** 👏🏽👏🏽

Hi @koomekevo , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Approved


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**